Packs/GoogleCloudSCC/README.md

```diff
@@ -0,0 +1,126 @@
+<p>
+
+1. Go to marketplace and search for <i><strong>Google Cloud Logging</i></strong>.<br>
+2. Install <i><strong>Google Cloud Logging</i></strong>.<br>
+3. Go to <i><strong>Data Sources</i></strong> and <i><strong>Add New Instance</i></strong><br>
+4. Connect <i><strong>Google Cloud Platform</i></strong> data source<br>
+5. Insert subscription name (Ingest logs and data from a GCP Pub/Sub section 3)<br>
+6. Insert credentials file (Ingest logs and data from a GCP Pub/Sub section 4)<br>
+7. Select <i><strong>Flow or Audit Logs</i></strong><br>
+8. Go to Query builder and use the dataset - google_cloud_logging_raw<br>
+
+</p>
+<h4> Note </h4>
+<p>
+To include audit logs related to Google Cloud Security Command Center only, you will need to add inclusion filter on the log router sink. <br>
+Add the filter <i><strong>protoPayload.serviceName="securitycenter.googleapis.com"</i></strong> <br>
+as describe in <a href="https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-and-data-from-a-GCP-Pub/Sub?tocId=RyZP~~W~YWbOPGKAClIQHg">section 2.c</a>. <br>
+</p>
+
+<a href="https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-and-data-from-a-GCP-Pub/Sub?tocId=RyZP~~W~YWbOPGKAClIQHg">Ingest logs and data from a GCP Pub/Sub</a>
+</details>
+<br>
+
+<h2>Sample Use Cases</h2>
+<p>
+1. <strong>Vulnerability findings - Public bucket ACL:</strong> When a cloud storage bucket is detected as publicly accessible (that means that anyone can read/ edit the content of the bucket) the user will notified about this event and will get a recommendation on how to act regarding this issue. <br> 
+usually for this type of event, you will need to remove users from the bucket's members list
+</p>
+<p>
+2. <strong>Vulnerability findings - Open RDP Port:</strong> When a firewall configuration is set to have an open RDP port that allows connections from all IP addresses on TCP | UDP port 3389. <br>
+The recommendation will be to restrict firewall rules.
+</p>
+<br>
+
+
+<h2>Configure Google Cloud Security Command Center</h2>
+ <p>
+To configure ingestion of data from Google Cloud Security Command Center follow the procedure below: <br>
+<a href="https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-and-data-from-a-GCP-Pub/Sub?tocId=RyZP~~W~YWbOPGKAClIQHg">Ingest logs and data from a GCP Pub/Sub</a>
+</p>
+<br> 
+ 
+
+<h2> Configure Cortex XSIAM </h2>
+<p>
+1. Go to marketplace and install Google Cloud SCC pack <br>
+2. Go to <i><strong>Data Sources</i></strong> and <i><strong>Add New Instance</i></strong> <br>
+3. Connect <i><strong>Google Cloud Platform</i></strong> data source<br>
+4. Insert subscription name (Ingest logs and data from a GCP Pub/Sub section 3)<br>
+5. Insert credentials file (Ingest logs and data from a GCP Pub/Sub section 4)<br>
```

Comment:
Not sure what you mean by section 3, section 4. Need a link.
I think you need to be more specific as to where it is or leave it out. Not sure it adds any additional information.